flocker/provision/_sphinx.py

```diff
@@ -66,7 +71,11 @@ def run(self):
         task = getattr(tasks, 'task_%s' % (self.arguments[0],))
         prompt = self.options.get('prompt', '$')
         if len(self.arguments) > 1:
-            task_arguments = self.arguments[1].split()
+            # Some tasks can include the latest installable version as (part
+            # of) an argument. This replaces a placeholder with that version.
+            latest = get_installable_version(version)
+            task_arguments = [item.replace(PLACEHOLDER, latest) for
+                              item in self.arguments[1].split()]
```

Comment:
The homebrew task does use the argument:

```python
def task_test_homebrew(recipe_url):
    return [
        Run(command="brew update"),
        Run(command="brew install {url}".format(url=recipe_url)),
        Run(command="brew test {url}".format(url=recipe_url)),
    ]
```

It uses the argument to install and test the recipe - the argument is named `recipe_url` but can really just be the name of the recipe if that repository is tapped. The argument is used in `commands = task_test_homebrew(recipe_url)` in `admin.testbrew.main`. The argument is also used in the new code:

`.. task:: test_homebrew flocker-|latest-installable|`

That task will render the commands from  `task_test_homebrew` as if the `recipe_url` were e.g. `flocker-0.3.3dev8` if 0.3.3dev8 is the latest installable package.